### PR TITLE
Added posibility to retry requests on failure

### PR DESCRIPTION
**Pull request by m-swat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,21 @@ Tests suite configuration consist of actions that are executed synchronously in
     }
     ```
     
+    If you want to retry request when error occurs, add `retry` key with the desired number of retries
+    
+    ```json
+    {
+        "type" : "request",
+        "retry": 2,
```
Comment (Flantive) [inline README.md]: retries

Comment (mmitis) [inline README.md]: please, change it to "retries".

File: lib/TestsRunner.js [hunk 2]
```diff
@@ -85,22 +85,40 @@ class TestsRunner {
...
             const options = this.fillTemplateWithData(config.options);
             const tests = this.fillTemplateWithData(config.tests);
+            const responseStackIndex = this.responseStack.length;
+            let retryCount = 0;
 
             this.printLog(`Running request to ${options.url}`);
 
-            request(options, (error, response) => {
+            const canRetry = () => {
+                return Boolean(config.retry && retryCount < config.retry);
```
Comment (mmitis) [inline lib/TestsRunner.js]: !!

**Review by mmitis — CHANGES_REQUESTED** [review]

**Review by Flantive — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (m-swat) [reply README.md]: done

**Review by Flantive — APPROVED** [review]

**Pull request merged**